browser-extensions.json

```diff
@@ -1,10 +1,10 @@
-{
-  "nordpass": {
+[
+  {
     "name": "NordPass",
     "message": "NordPass performs polyfill-like transformations on navigator.credentials, affecting tests for the CredentialsContainer API.",
     "affectedTests": ["api.CredentialsContainer"],
     "chromeId": "eiaeiblijfjekdanodkjadfinkhbfgcd",
     "firefoxId": "0ff38d7a-db91-4a1a-a222-00f10d098b22",
     "safariId": "9F9F9D62-DE32-4B4F-88FD-63D448DACFDC"
   }
-}
+]
```

Comment:
We need to retain keys in this file because the keys are reported in the test results.  Can we revert this and use `for key in dict` logic instead?